### PR TITLE
Soft crit, except not terrible

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: While below 0 health but above -30 health, you will be able to crawl slowly if not pulled, whisper, hear speech, and see with worsening vision.
wip: The previous softcrit got reverted, for changelog context.
/:cl:

Bonuses: You can crawl! You'll still die instead of being in limbo! You can hear your murderer gloat! You can whisper for help!
Negatives: You'll die! You can't do items or item things! If you're being pulled, you can't crawl! Crawling is slow as shit!

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -604,50 +604,51 @@
...
 		switch(health)
-			if(-20 to -10) severity = 1
-			if(-30 to -20) severity = 2
-			if(-40 to -30) severity = 3
-			if(-50 to -40) severity = 4
-			if(-60 to -50) severity = 5
-			if(-70 to -60) severity = 6
-			if(-80 to -70) severity = 7
-			if(-90 to -80) severity = 8
+			if(-20 to -10) severity = 2
```
Comment (lzimann) [inline code/modules/mob/living/carbon/carbon.dm]: new line on these `severity` please

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -604,50 +604,51 @@
...
-				if(85 to INFINITY) severity = 6
-			overlay_fullscreen("brute", /obj/screen/fullscreen/brute, severity)
-		else
-			clear_fullscreen("brute")
+
+	//Oxygen damage overlay
+	if(oxyloss)
+		var/severity = 0
+		switch(oxyloss)
+			if(10 to 20) severity = 1
```
Comment (lzimann) [inline code/modules/mob/living/carbon/carbon.dm]: same here

**Comment by PKPenguin321** [comment]
Honestly works better without being able to whisper over the radio imo

Bug: If you are in soft crit and dip into full crit from oxy damage and are healed back to soft crit, you keep the blind overlay

**Comment by ChangelingRain** [comment]
Fixed both of those; you can't radio in crit, and the blind overlay should get removed.

**Comment by PKPenguin321** [comment]
This is balance/rebalance and feature more than tweak but yeah it's neat 👍

**Comment by CosmicScientist** [comment]
Can I use an elaborate voice analyser setup to sound the alarm for the HoS or AI to hear?

**Comment by lzimann** [comment]
You can already do this with the current crit/death whisper

**Comment by optimumtact** [comment]
I think the feature tag fits best, since it is new functionality

**Comment by Anonmare** [comment]
@CosmicScientist  I'm fairly certain whispers can't be heard over radios so you couldn't do that

You could use a health sensor and signaller assembly to trigger on a custom frequency and give the HoS a corresponding signaller. If it goes off, he'd know you're in trouble

**Comment by CosmicScientist** [comment]
The health sensor sounds like a better idea... actually I'm wondering why I don't do that more often. That's actually a cool idea.

**Comment by BeeSting12** [comment]
From #30264 
"Round ID: 75380

I was the Warden (Ollanius Pius) trying to hunt down a traitor with a .357 revolver. Multiple shots got me into crit and I was dying normally. A few people patched me up enough for me to be able to hear (the new "soft crit" mechanic), but I also found myself able to TALK in soft crit.

When I was healed back to 100% health, I was still blinded and lying on the floor, unable to get up or get helped to get back up again. I had no other reagents on my body. The only solution was to monkeyfy me and make me back human (thanks to admin intervention, thanks Duncdar!).

Please solve this ASAP as it's extremely game breaking."

**Comment by YakumoChen** [comment]
still buggy as shit, I could move while critted for a bit, couldn't ghost out correctly (had to manually press the button) and was in crit for like 10 centuries because the oxyloss wasn't proccing

**Comment by Togopalis** [comment]
People in cryotubes are registered as dead while still alive

**Comment by ChangelingRain** [comment]
First two should be fixed with the new round, third should be fixed since I redid the TGUI

**Comment by KorPhaeron** [comment]
Does crawling leave a blood trail like it does when you are dragged/bleeding? If not it probably should if that isn't too difficult

Distinction in logging between a succumb in hard crit/soft crit would be nice

Finally one of the things I was hoping for with this was a way for people to incapacitate without stuns and without it appearing like murder to admins, and having them take regular crit damage takes away from that. I understand people don't like indefinite soft crit but oxyloss should at least be a bit slower than regular crit (and obviously it will speed up if you start dragging yourself and losing blood)

**Comment by ChangelingRain** [comment]
Except for slower crit damage, which can't be easily done well, I've done that!

**Comment by KorPhaeron** [comment]
🎉

**Comment by kevinz000** [comment]
yay now i can say "nothin personal kid" like an edgy fucker AFTER I crit my victim!

**Comment by FantasticFwoosh** [comment]
Now to spam `OH GOD THE PAIN` every two seconds and its the full bay experience of being not quite dead, but everyone around you is too incompetent/rule restricted from healing you.

**Comment by factoryman942** [comment]
under the old softcrit pr people in crit showed as dead in both sleepers and cryo

if they're showing dead in cryo with this pr they probably show as dead in sleepers too

**Comment by Cyberboss** [comment]
@ChangelingRain probably stuff passing stat to tgui and tgui has to use the raw numbers instead of defines

**Comment by ChangelingRain** [comment]
I know and fixed it already did you not look at commit [#b0d7bde](https://github.com/tgstation/tgstation/pull/30262/commits/b0d7bde80cd44fdc57306fb71b8916ae6fc6b861)

**Comment by DaxDupont** [comment]
The following issues currently exist with cryo:
It sometimes will not turn on, will just turn off
It will report no occupant
Temperature will sometimes be NaN even with occupant

I used it as a borg

**Comment by Cyberboss** [comment]
https://github.com/tgstation/tgstation/blob/master/code/modules/mob/mob.dm#L440

**Comment by Robustin** [comment]
How does this softcrit work with deathgasps? Do I have to whisper to talk or will my talking automatically come out as whispers? If I talk while in DEEP CRIT will I not say anything or will I deathgasp?

**Comment by ChangelingRain** [comment]
After the fix, you'll whisper automatically and in DEEPCRIT you won't autowhisper and thus won't deathgasp.

**Comment by Steelpoint** [comment]
Question. How long does it take to naturally accumulate 30 oxygen damage over time when in crit?

**Comment by KorPhaeron** [comment]
I played earlier this morning and people were still yelling on the radio in soft crit

**Comment by KorPhaeron** [comment]
Antur reviewed it this morning and its been okay in the most recent tests as far as I know

**Pull request merged**